File: 23_C#Basics/_04_LoopsPartOneMoreExercises/_07_FootballLeague/_07_FootballLeague.cs
using System;

public class _07_FootballLeague
{
    public static void Main(string[] args)
    {
        // Read input values
        int stadiumCapacity = int.Parse(Console.ReadLine());
        double fansCount = double.Parse(Console.ReadLine());

        // Add variables
        double sectorA = 0;
        double sectorB = 0;
        double sectorV = 0;
        double sectorG = 0;

        // Process each fan's sector and calculate statistics
        for (int i = 0; i < fansCount; i++)
        {
            string sector = Console.ReadLine().ToLower();

            if (sector == "a")
            {
                sectorA++;
            }
            else if (sector == "b")
            {
                sectorB++;
            }
            else if (sector == "v")
            {
                sectorV++;
            }
            else if (sector == "g")
            {
                sectorG++;
            }
        }

        // Calculate and print the statistics
        Console.WriteLine("{0:F2}%", sectorA / fansCount * 100);
        Console.WriteLine("{0:F2}%", sectorB / fansCount * 100);
        Console.WriteLine("{0:F2}%", sectorV / fansCount * 100);
        Console.WriteLine("{0:F2}%", sectorG / fansCount * 100);
        Console.WriteLine("{0:F2}%", fansCount / stadiumCapacity * 100);
    }
}

File: 01_ProgrammingBasics/_04_LoopsPartOneMoreExercises/_07_FootballLeague.java
package _01_ProgrammingBasics._04_LoopsPartOneMoreExercises;

import java.util.Scanner;

public class _07_FootballLeague {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        int stadiumCapacity = Integer.parseInt(scan.nextLine());
        double fansCount = Double.parseDouble(scan.nextLine());

        // Add variables
        double sectorA = 0;
        double sectorB = 0;
        double sectorV = 0;
        double sectorG = 0;

        // Process each fan's sector and calculate statistics
        for (int i = 0; i < fansCount; i++) {
            String sector = scan.nextLine().toLowerCase();

            if (sector.equals("a")) {
                sectorA++;
            } else if (sector.equals("b")) {
                sectorB++;
            } else if (sector.equals("v")) {
                sectorV++;
            } else if (sector.equals("g")) {
                sectorG++;
            }
        }

        // Calculate and print the statistics
        System.out.printf("%.2f%%%n", sectorA / fansCount * 100);
        System.out.printf("%.2f%%%n", sectorB / fansCount * 100);
        System.out.printf("%.2f%%%n", sectorV / fansCount * 100);
        System.out.printf("%.2f%%%n", sectorG / fansCount * 100);
        System.out.printf("%.2f%%%n", fansCount / stadiumCapacity * 100);

        scan.close();
    }
}
